### PR TITLE
Upgrade to Delta 2.1.0 and Spark 3.3

### PR DESCRIPTION
**Pull request by himanishk**
- Upgrade to Delta 2.1.0 and Spark 3.3
- Switch to using AvailableNow as default Trigger interval
- Documentation updates

### CODE REVIEW

**Review by aemrob — APPROVED** [review]
LGTM

**Review by aemrob — APPROVED** [review]
LGTM

**Comment by himanishk** [comment]
@alexott @nfx - @aemrob has looked into these changes and approved them. Unfortunately, the PR still shows _1 approving review is required_ . Maybe we do not have the correct permissions. 

Could you please approve the changes for me to MERGE these changes and make a release. Thanks

File: build.sbt [hunk 1]
```diff
@@ -102,10 +103,14 @@ lazy val root = (project in file(".")).
     buildInfoPackage := "com.databricks.labs.deltaoms.common",
     assembly / test  := {},
     assembly / assemblyOption := (assembly / assemblyOption).value.copy(includeScala = false),
-    assembly / assemblyShadeRules := Seq(
-      ShadeRule.rename("org.apache.spark.sql.delta.**" ->
-        "com.databricks.sql.transaction.tahoe.@1").inAll
-    ),
+    if (buildTarget == "db") {
```
Comment (nfx) [inline build.sbt]: Make it only-Databricks target 😉 it must not run outside of Databricks.

File: docs/content/faq/general.md [hunk 2]
```diff
@@ -50,15 +50,33 @@ For example, it could answer questions like :
 Data Engineering teams, Data Lake Admins and Operational Analysts would be able to 
 manage and use this feature for operational insights on the Delta Lake. 
 
+**Q Can I run this solution on non-Databricks environment ?**
```
Comment (nfx) [inline docs/content/faq/general.md:53]: The answer must be "no", because of Databricks License this project is distributed by.

**Review by nfx — CHANGES_REQUESTED** [review]
Please adjust the code and instructions to be compatible with Databricks License

File: build.sbt (hunk 1 above)
Comment (himanishk) [reply build.sbt]: @nfx , ok. Made the changes

File: docs/content/faq/general.md (hunk 2 above)
Comment (himanishk) [reply docs/content/faq/general.md:53]: @nfx , ok. Made the changes

**Review by nfx — APPROVED** [review]

**Pull request merged**